recipes/streamtracer/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "streamtracer" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamtracer-{{ version }}.tar.gz
+  sha256: 29955653f88970ab3b1fe3e9c470e3b243c1a04e157647d22b2b13494f8864ca
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - numpy
+    - maturin >=0.13,<0.14
+    - pip
+  run:
+    - python
+    - numpy >=1.14.5
+    - libgcc-ng  # [linux]
+
+test:
+  imports:
+    - streamtracer
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://streamtracer.readthedocs.io/en/latest/
+  summary: Python wrapped fortran to calculate streamlines
```

Comment:
Fortran? Is this summary correct? This recipe is using rust, no?

Reply:
I'm not the maintainer of the original code (but in contact), so I did not change any description from the original one. But yes, I think it was written in Fortran and is in the process of being moved to Rust.